src/FunctionalTestCase.php

```diff
@@ -13,6 +13,10 @@ abstract class FunctionalTestCase extends WebTestCase
 {
     protected static function loadFixtures(array $yamlFiles, bool $truncateTables = true): void
     {
+        if (!self::$booted) {
```

Comment:
Are you sure that this works? The `$booted` property is defined in the `KernelTestCase` class, so we might have to use `static::` instead of `self::`.

Reply:
As far as I know, properties are inherited to the current class. I would need to call `static` if I want the property (or method) of my _child classes_, not my parent classes.

Reply:
As long as the `$booted` property does not get redeclared in the child classes `parent::`, `self::` and `static::` refer to the same property.